snippets/olmv1-known-issue-private-registries.adoc

```diff
@@ -0,0 +1,15 @@
+// Text snippet included in the following modules:
+//
+// * release_notes/ocp-4-17-release-notes.adoc (enterprise-4.17 branch only)
+// * extensions/arch/catalogd.adoc
+// * extensions/catalogs/creating-catalogs.adoc
+// * extensions/catalogs/fbc.adoc
+// * extensions/catalogs/managing-catalogs.adoc
+// * extensions/catalogs/rh-catalogs.adoc
+// * extensions/ce/managing-ce.adoc
+// * extensions/ce/upgrade-edges.adoc
+// * extensions/index.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+Currently, {olmv1-first} cannot authenticate private registries, such as the Red Hat-provided Operator catalogs. This is a known issue. As a result, the {olmv1} procedures that rely on having the Red Hat Operators catalog installed do not work. (link:https://issues.redhat.com/browse/OCPBUGS-36364[*OCPBUGS-36364*])
```

Comment:
@michaelryanpeter just FYI: it is for clusterextension which does not support pull image from private registry.
for clustercatalog, we can create secret to pull image from private registry, even for registry.redhat.io.

Reply:
@kuiwang02 I created a ticket to revise this note after GA with a better description: https://issues.redhat.com/browse/OSDOCS-12125

Thank you!